BasicClasses/BasicClasses.tex

```diff
@@ -914,27 +914,21 @@ \section{Exceptions}
 However, exception handlers have a couple of other options for dealing with exceptions by sending messages to them:
 
 \begin{itemize}
-    \item \mthind{Exception}{return:} returns a value from the handler block.
-    This is the implicit default behavior of every handler block.
-    For instance, the following  would be equivalent to the first example above:
-
-    \begin{code}{@TEST}
-    [(-2 to:2) collect: [:i | 100 / i]]
-       on: ZeroDivide
-       do: [:ex | ex return: nil].
-    \end{code}
-
     \item \mthind{Exception}{pass} passes the exception to the next handler on the stack\footnote{This is similar to a plain \ct{throw} in other languages.} (or to the \mthind{Exception}{defaultAction}, if there is no further exception handler).
+    Unlike sending \mthind{Exception}{signal} again to the exception, \ct{pass} will maintain the existing signaler stack, which is usually desired behavior (as you can debug and resume from the exception).
```

Comment:
"from the exception" -> "from the original exception"?

very minor

Reply:
```suggestion
    Unlike sending \mthind{Exception}{signal} again to the exception, \ct{pass} will maintain the existing signaler stack, which is usually desired behavior (as you can debug and resume from the original exception).
```